CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+enable_testing()
+find_library(Qiskit qiskit_c_ext PATHS target/release REQUIRED)
```

Comment:
How does this know that the library header ist `qiskit.h` in lowercase? 🤔

Reply:
Ahh true (you can tell it's been 8 years I used CMake 😛), thanks! Then `Qiskit` is just the variable name you give the found lib, right?